resources/js/components/sections/bookshelfAddStep1.vue

```diff
@@ -33,8 +34,16 @@ export default {
   data () {
     return {
       searchText: ''
+      
+
     }
   },
+  beforeDestroy(){
+    this.reset()
+
+  }
+  
+  ,
```

Comment:
please remove extra lines and make sure the comma is on the same line as the `}`